rotkehlchen/globaldb/updates.py

```diff
@@ -434,6 +434,12 @@ def _handle_asset_update(
 
         try:
             with connection.savepoint_ctx() as cursor:
+                if action.startswith('UPDATE') and cursor.execute(
```

Comment:
I would do that strip thing to avoid issues with leading spaces

Reply:
Do we ever delete from the assets? Is it something that we need to take care?

Reply:
I don't think we support deleting. I just checked that for fixing the asset collections in v25. The fallback queries are regex matched for INSERT queries only.

adding the .strip()